.github/workflows/playwright-pr.yml

```
@@ -0,0 +1,136 @@
+      - '**.tsx'
+      - 'package*.json'
+      - 'playwright.config.js'
+      - '.github/workflows/playwright*.yml'
+
+jobs:
+  # Quick smoke tests for PRs
+  smoke-test:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)